tpot/base.py

```diff
@@ -727,7 +727,7 @@ def _wrapped_cross_val_score(sklearn_pipeline, features=features, classes=classe
                 resulting_score = -float('inf')
             return resulting_score
 
-        if not sys.platform.startswith('win'):
+        if self.n_jobs != 1:
```

Comment:
It seems that there's a lot of repeated code between the two branches of this `if` statement. Is it possible to instead make the `if` statement decide what job mapper to use (1), then the rest of the code can be the same?

(1) It will use the job pool if n_jobs != 1, otherwise it will use regular `map`?